packages/cli/lib/utils.js

```diff
@@ -96,3 +96,11 @@ export function copyDirRecursive(from, to, exclude) {
     }
   });
 };
+
+/**
+ * check Operating System
+ * @returns {boolean}
+ */
+export function isWin() {
+  return process.platform === "win32" ? true : false;
+}
```

Comment:
`utils.js` might be renamed `utils/isWin32.js` and the ternary seems redundant when the equality statement alone can be the return value, for instance
```suggestion
/**
 * Check if process is running on Windows 32-bit operating system
 * @returns {boolean}
 */
export function isWin32 = () => process.platform === "win32";
}
```